cita-auth/src/main.rs

```diff
@@ -83,7 +82,7 @@ extern crate serde_derive;
 extern crate tempfile;
 #[macro_use]
 extern crate util;
-extern crate db as cita_db;
+extern crate cita_database;
```

Comment:
You don't need to `extern` the crate in Edition 2018?

Reply:
DONE